vortex-layout/src/layouts/struct_/eval_expr.rs

```diff
@@ -23,18 +23,9 @@ use crate::{
 impl ExprEvaluator for StructReader {
     fn pruning_evaluation(
         &self,
-        row_range: &Range<u64>,
-        expr: &ExprRef,
+        _row_range: &Range<u64>,
+        _expr: &ExprRef,
     ) -> VortexResult<Box<dyn PruningEvaluation>> {
-        // Partition the expression into expressions that can be evaluated over individual fields
-        let partitioned = self.partition_expr(expr.clone());
-
-        if partitioned.partition_names.len() == 1 {
```

Comment:
Hmm, for pruning this may actually be important...

Reply:
I only included this change since you suggested it. Let's remove it for now